src/actions/helpers/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const ETHEREUM_API_ID = "ethereum";
+export const ERC20_PRICE_ENDPOINT =
+  "https://api.coingecko.com/api/v3/simple/token_price/ethereum?";
+export const ETHEREUM_PRICE_ENDPOINT =
```

Comment:
These constants aren't needed to be exposed elsewhere in the outward facing SDK. They could possibly be put in the configurations that exist for each network and the top level `Config` object, but it would give them more visibility than required